Rakefile

```
@@ -1,11 +1,11 @@
 Dir.glob("lib/tasks/**/*.rake").each { |r| load r }
 
 desc "Build the static site"
-task :build => ["contributors:update", :repo_pages, :man] do
+task build: %w[contributors:update repo_pages man] do
```

Can we also enable the cop to enforce `%w[]` for Arrays of Strings?

---

We can do it in another PR.

---

Or, I can revert this part to the following in this PR:

```suggestion
task build: ["contributors:update", :repo_pages, :man] do
```

---

Reverted.